### PR TITLE
fix: install script with specification

### PR DESCRIPTION
**Pull request by MakarenkoAI**

### CODE REVIEW

File: specification.scs [hunk 1]
```diff
@@ -14,10 +14,8 @@ sc_web
     => nrel_explanation: [SC-Web is an intelligent knowledge base user interface] (*<- lang_en;; *);
     => nrel_component_dependencies: ... (* <- empty_set;; *);
 
-    => nrel_installation_script: [/scripts/install_deps_ubuntu.sh](* <- lang_en;; *);
```
Comment (MksmOrlov) [inline specification.scs:17]: remove lang_en in script formalization

File: scripts/install.sh [hunk 2]
```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+set -eo pipefail
+
+CURRENT_DIR=$(cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd)
+BLUE='\033[0;34m'
+NC='\033[0m'
+
+echo -e "${BLUE}Installation\v${NC}"
```
Comment (MksmOrlov) [inline scripts/install.sh:8]: can we use `stage` instead of echo?

File: scripts/install.sh [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+"${CURRENT_DIR}/install_deps_ubuntu.sh"
+
+echo -e "${BLUE}\vDeps\v${NC}"
+
+pip3 install -r "${CURRENT_DIR}/../requirements.txt" && npm install
```
Comment (MksmOrlov) [inline scripts/install.sh:14]: install_deps_ubuntu already execute this

**Review by MksmOrlov — CHANGES_REQUESTED** [review]

File: scripts/install.sh (hunk 2 above)
Comment (MakarenkoAI) [reply scripts/install.sh:8]: no

File: specification.scs [hunk 4]
```diff
@@ -14,10 +14,7 @@ sc_web
...
-    => nrel_installation_script: [/scripts/install.sh](* <- lang_en;; *);
-
-    => nrel_component_address: [https://github.com/ostis-ai/sc-web] (* <- concept_github_url;; *);
+    => nrel_installation_script: [/scripts/install.sh];
+    => nrel_component_address: [https://github.com/MakarenkoAI/sc-web] (* <- concept_github_url;; *);
```
Comment (MksmOrlov) [inline specification.scs:18]:
```suggestion
    => nrel_component_address: [https://github.com/ostis-ai/sc-web] (* <- concept_github_url;; *);
```

File: scripts/install.sh [hunk 5]
```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+set -eo pipefail
+
+CURRENT_DIR=$(cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd)
+source "${CURRENT_DIR}/formats.sh"
+
+stage "Installation"
+
+"${CURRENT_DIR}/install_deps_ubuntu.sh"
```
Comment (MksmOrlov) [inline scripts/install.sh:9]:
```suggestion
"${CURRENT_DIR}/install_dependencies.sh"
```

File: scripts/install.sh [hunk 6]
```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+set -eo pipefail
+
+CURRENT_DIR=$(cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd)
+source "${CURRENT_DIR}/formats.sh"
+
+stage "Installation"
```
Comment (MksmOrlov) [inline scripts/install.sh:7]:
```suggestion
stage "Install depemdencies"
```

File: scripts/install.sh [hunk 7]
```diff
@@ -0,0 +1,11 @@
...
+
+"${CURRENT_DIR}/install_deps_ubuntu.sh"
+
```
Comment (MksmOrlov) [inline scripts/install.sh:10]: stage "sc-web installed successfully"

**Review by MksmOrlov — CHANGES_REQUESTED** [review]

**Review by MksmOrlov — APPROVED** [review]

**Review by NikitaZotov — APPROVED** [review]

**Pull request merged**